op-chain-ops/genesis/setters.go

```diff
@@ -14,6 +14,26 @@ import (
 	"github.com/ethereum/go-ethereum/log"
 )
 
+// UntouchablePredeploys are addresses in the predeploy namespace
+// that should not be touched by the migration process.
+var UntouchablePredeploys = map[common.Address]bool{
+	predeploys.LegacyMessagePasserAddr: true,
+	predeploys.DeployerWhitelistAddr:   true,
+	predeploys.GovernanceTokenAddr:     true,
+	predeploys.L1BlockNumberAddr:       true,
+	predeploys.WETH9Addr:               true,
+}
+
+// UntouchableCodeHashes contains code hashes of all the contracts
+// that should not be touched by the migration process.
+var UntouchableCodeHashes = map[common.Address]common.Hash{
```

Comment:
Did you pull these code hashes from goerli? There exists the possibility that mainnet has slightly different code. Unlikely but it's possible because I can't remember if the compiler version got changed

Reply:
I pulled from Goerli. Will check mainnet.

Reply:
Updated to pull form both Goerli and mainnet.